### ISSUE TITLE
StripeiOS-Static framework compiled for iOS 8, not iOS 7

### ISSUE THREAD

**Issue by rweichler** [stripe/stripe-ios#785]
Linking to that framework yields a bunch of these types of errors:

```
ld: warning: object file (deps/frameworks/Stripe.framework/Stripe(NSBundle+Stripe_AppName.o)) was built for newer iOS version (8.0) than being linked (7.0)
```

In the [docs](https://stripe.com/docs/mobile/ios), it says "Note: We only recommend this approach if you need to support iOS 7 in your application." with regards to the static framework.

**Comment by rweichler** [comment]
I'm talking about the one in the releases page btw: https://github.com/stripe/stripe-ios/releases/tag/v11.2.0

**Comment by joeydong-stripe** [comment]
Hi @rweichler,

Sorry for the confusion, we were in the middle of adjusting our docs for iOS 7 support.

Is your app targeting iOS 7? If so, can you try linking to our last supported version for iOS 7 – v8.0.7? 

https://github.com/stripe/stripe-ios/releases/tag/v8.0.7

**Comment by rweichler** [comment]
I patched 11.2.0 to be compatible with iOS 7. Was just letting you guys know that there's a discrepancy.

**Comment by joeydong-stripe** [comment]
Okay sounds good! We're on it :)
